src/docs/reference/modules/AuditTrail/README.md

```diff
@@ -0,0 +1,58 @@
+# Audit Trail (`OrchardCore.AuditTrail`)
+
+The Audit Trail module provides an immutable (for users, even administrators but not for developers), auditable log of certain changes and events in the system. This includes e.g. creation or deletion of content items, and events like user login failures. For content items, previous versions and deleted items can be restored, changes tracked.
+
+## General Concepts
+
+Audit Trail events are created if a certain supported event happened in the system. While there are several built-in event handlers for the most important events, this is extensible and module authors can provide their own ones.
+
+Once you've enabled the module, you will see a new menu item in the admin UI called *Audit Trail*. The list there shows you the logged Audit Trail events.
+
+## Using the Audit Trail event list
+
+The Audit Trail event list provides you with filtering and pagination to be able to easily find the Audit Trail events that you are looking for. By default, you can filter by the following criteria:
```

Comment:
The Audit Trail event list provides you with filtering and pagination to be able to find the Audit Trail events that you are looking for easily.

Reply:
If you want to make it easier to see the difference, please add such suggestions as code suggestions :)

![image](https://user-images.githubusercontent.com/1976647/153884070-da66e272-bf96-44aa-a6e2-70aeb0466347.png)